storage-manager/src/S3Storage.cpp

```diff
@@ -210,19 +215,27 @@ S3Storage::S3Storage(bool skipRetry) : skipRetryableErrors(skipRetry)
   }
 
   endpoint = config->getValue("S3", "endpoint");
+  if (!connect_timeout.empty())
+  {
+      connectTimeout = stof(connect_timeout);
+  }
+  if (!timeout.empty())
+  {
+      operationTimeout = stof(timeout);
+  }
 
   ms3_library_init();
   if (libs3_debug == "enabled")
   {
-    ms3_debug();
+    ms3_debug(1);
```

Comment:
Really? It didn't work previously?

Reply:
Previously libmarias3 submodule pointed to a commit before that API change